comps/lvms_pg/README.md

```diff
@@ -0,0 +1,52 @@
+# LVM Prediction Guard Microservice
+
+Visual Question and Answering is one of the multimodal tasks empowered by LVMs (Large Visual Models). This microservice supports visual Q&A by using a LLaVA model available via the Prediction Guard API. It accepts two inputs: a prompt and an image. It outputs the answer to the prompt about the image.
+
+# 🚀1. Start Microservice with Python
+
+## 1.1 Install Requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+## 1.2 Start LVM Service
+
+```bash
+python lvm.py
+```
+
+# 🚀2. Start Microservice with Docker (Option 2)
+
+## 2.1 Setup Environment Variables
+
+Setup the following environment variables first
+
+```bash
+export PREDICTIONGUARD_API_KEY=${your_predictionguard_api_key}
+```
+
+## 2.1 Build Docker Images
+
+```bash
+cd ../..
+docker build -t opea/lvm-pg:latest -f comps/lvms_pg/Dockerfile .
```

Comment:
I wonder if they have CI/CD in place to automate the release of these docker images to the `opea` org on docker hub? Or if we should host them. Might be good to check.

Reply:
This may be a good question to ask when we open a PR against the original repo.